microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
```

Comment:
This persistent_storage_microshift directory doesn't have any of the 3 symlinks in it